rssfeed/models.py

```diff
@@ -17,37 +18,27 @@ def __unicode__(self):
         return self.title or self.xml_url
 
     def save(self, *args, **kwargs):
-        """Poll new Feed"""
+        """Poll new Feed """
```

Comment:
I think @cobusc has more insight into how we handle doc strings.

But primarily we use PEP8 as the baseline

```
"""Text
"""

"""
Text
"""
```